Script/peopleController.js

```diff
@@ -31,7 +31,7 @@ function loadAssignee()
     if (response.data.length == 0) return;
 
     var res = response.data;
-    $("#assignee").html(res[0].forename + " " + res[0].surname);
+    $("#assignee").html(res[0].forename + " " + res[0].surname + " " + "(" + res[0].username + ")");
```

Comment:
Use Template Literals like in the pro video, check the URL for better understanding
https://blog.webdevsimplified.com/2020-03/tagged-template-literals/